install/relay-credentials.sh

```diff
@@ -14,7 +14,7 @@ if [[ ! -f "$RELAY_CREDENTIALS_JSON" ]]; then
   # creating an empty credentials file before relay runs.
 
   $dcr \
-    --no-deps \
+    --no-deps -T \
```

Comment:
Nit: Can we use the long form `--no-TTY` for better readability?

Reply:
Done.